build.gradle

```diff
@@ -3,7 +3,7 @@ buildscript {
         kotlin_version = '1.6.10'
         coroutines_version = '1.5.2'
         ok_http_version = '4.9.1'
-        dashjVersion = '0.18.0'
+        dashjVersion = '0.18.1-SNAPSHOT'
```

Comment:
We won't need this commit in #1035 if it is merged after 8.0.1 is released and merged to master.  Those changes will include this change.